test/rebar3_hex_integration_SUITE_data/test_apps/doc_dir_missing_index/.gitignore

```diff
@@ -0,0 +1,19 @@
+.rebar3
```

Comment:
This is lazy, but I was having trouble getting the path to the test app and wanted to get this in quick for re-publishing 6.11.0

Reply:
Switched to filelib:is_file/1, this should be solid now I believe.